analysis/runtime/src/mir_loc.rs

```diff
@@ -169,6 +169,8 @@ pub struct EventMetadata {
     pub destination: Option<MirPlace>,
     /// Destination func [`DefPathHash`] of [`Event`](crate::events::Event).
     pub transfer_kind: TransferKind,
+    /// Any string useful for debugging
+    pub debug: String
```

Comment:
Maybe call this `debug_info` instead.

Reply:
Fixed in https://github.com/immunant/c2rust/pull/566/commits/a9fd15ff6a20f7e05758ed8aa83743b9ba6bfb77.